src/Hydrator/MoneyHydrator.php

```diff
@@ -20,7 +20,7 @@ class MoneyHydrator implements HydratorInterface
     public function extract($object)
     {
         return [
-            'amount' => $object->getAmount(),
+            'amount' => $object->getAmount() / 100,
```

Comment:
@prolic @danizord
-  ok for testing evangelista, null could be tested for. 
- form elem how implements this one i hope brazil. (pls using of attribute "step" should be skipped on config maybe)